easybuild/easyconfigs/g/GPAW/GPAW-19.8.1-intel-2019a-ASE-3.18.0-Python-3.7.2.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = "PythonPackage"
+
+name = 'GPAW'
+version = '19.8.1'
+aseversion = '3.18.0'
+pythonsuffix = '-Python-%(pyver)s'
+versionsuffix = '-ASE-' + aseversion + pythonsuffix
+
+homepage = 'https://wiki.fysik.dtu.dk/gpaw/'
+description = """GPAW is a density-functional theory (DFT) Python code based on the projector-augmented wave (PAW)
+ method and the atomic simulation environment (ASE). It uses real-space uniform grids and multigrid methods or
+ atom-centered basis-functions."""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+patches = [
+    ('GPAW-19.8.1-customize-intel.patch', 0),
+]
+checksums = [
+    '79dee367d695d68409c4d69edcbad5c8679137d6715da403f6c2500cb2178c2a',  # gpaw-19.8.1.tar.gz
+    '53f3f1b3ec487d7cc0981f5f2fbb6bb647453be6c2b7b2eacd468310bce75020',  # GPAW-19.8.1-customize-intel.patch
+]
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('SciPy-bundle', '2019.03'),
+    ('ASE', aseversion, pythonsuffix),
+    ('libxc', '4.3.4'),
```

Comment:
Is there a reason to leave out libvdw for intel?

Reply:
Yes, libvdwxc requires an MPI FFT function that is in FFTW but absent in the Intel MKL.  And having both FFTW and MKL linked together causes core dumps (reportedly they both implement helper functions with the same name but with different behaviour).

Users who need the libvdwxc functionality in GPAW will have to use the (slower) foss version.